src/views/Questionnaire/SmartApp.tsx

```diff
@@ -614,6 +640,19 @@ export function SmartApp(props: SmartAppProps) {
               ref={onFilterCheckboxRefChange}
             ></input>
           </div>
+          { showRequiredCheckbox ? 
+            <div className="task-button">
+              <label>Ignore required fields</label>{' '}
+              <input
+                type="checkbox"
+                onChange={() => {
+                  updateRequired(false);
+                }}
+                id='required-fields-checkbox'
```

Comment:
the id needs to be questionnaire specific, otherwise in the tab view there will be confusion about what state the required checkbox should be in between different questionnaire tabs.